src/components/NavBar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {
+  Link,
+} from 'react-router-dom';
+
+const NavBar = () => (
+  <div className="header">
+    <h1>Bookstore CMS</h1>
+    <ul className="nav-links">
+      <li className="link"><Link to="/">BOOKS</Link></li>
+
+      <li className="link"><Link to="/category">CATEGORY</Link></li>
+    </ul>
+  </div>
```

Comment:
![remove-margin](https://user-images.githubusercontent.com/93039359/193423022-0b2e68d3-07d2-4ba5-82d4-38ca637295ef.png)
- [ ] Kindly remove the margin-top from your navbar as you can see in the above image with red lines :+1: to follow the design :blush: